frontend/public/components/image-stream-tag.tsx

```diff
@@ -122,9 +126,13 @@ export const ImageStreamTagsDetails: React.SFC<ImageStreamTagsDetailsProps> = ({
   </div>;
 };
 
+const parseName = (nameAndTag: string): string => {
+  return nameAndTag.split(':')[0];
+};
+
 export const getImageStreamNameForTag = (imageStreamTag: K8sResourceKind): string => {
   const name = _.get(imageStreamTag, 'metadata.name', '');
-  return name.split(':')[0];
+  return parseName(name);
```

Comment:
It's a shame we have to parse the tag name to get the image stream name, but I don't see another way :/